web/src/components/feature/PhaseTable.tsx

```diff
@@ -31,6 +31,7 @@ export const PhaseTable = (props: {
         'Mercury Flow (g/d)',
       ]}
       rows={getRows(multiFlashResponse, cubicFeedFlow)}
+      density={'comfortable'}
```

Comment:
Do we want the tables to have different density?

Reply:
According to the sketches they should